aya/src/sys/bpf.rs

```diff
@@ -1056,6 +1057,20 @@ pub(crate) fn iter_map_ids() -> impl Iterator<Item = Result<u32, SyscallError>>
     iter_obj_ids(bpf_cmd::BPF_MAP_GET_NEXT_ID, "bpf_map_get_next_id")
 }
 
+pub(crate) fn bpf_enable_stats(bpf_stats_type: bpf_stats_type) -> Result<OwnedFd, SyscallError> {
+    let mut attr = unsafe { mem::zeroed::<bpf_attr>() };
+    attr.enable_stats.type_ = bpf_stats_type as u32;
+
+    // SAFETY: BPF_ENABLE_STATS returns a new file descriptor.
+    unsafe { fd_sys_bpf(bpf_cmd::BPF_ENABLE_STATS, &mut attr) }.map_err(|(code, io_error)| {
+        assert_eq!(code, -1);
```

Comment:
I was mainly following some of the other bpf syscall examples, where they also had this assert. I assumed it was to check for `-EPERM`, which is returned when privileges/capabilities are not met. But I can remove it since it's not necessary.